docker-compose.override.yml

```diff
@@ -15,8 +15,25 @@ services:
   web:
     build:
       target: development
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    environment:
+      - MINIO_HOST=localhost:9000
+      - MINIO_ACCESS_KEY=normal-oj-testing
+      - MINIO_SECRET_KEY=normal-oj-testing-secret
+      - MINIO_BUCKET=normal-oj-testing
     volumes:
       - ./Back-End:/app
+  minio:
+    image: quay.io/minio/minio
```

Comment:
maybe we can pin image version?

Reply:
you mean like `quay.io./minio/minio:RELEASE.2025-04-22T22-12-26Z`?

Reply:
y

Reply:
thanks, fixed.